### DISCUSSION TITLE
Can I do this on a raspberry pi 4b

### DISCUSSION THREAD

**Discussion by gauravdesale8** [jomjol/AI-on-the-edge-device#2985] [General]
I have a Raspberry Pi 4B and a Raspberry Pi Camera. Since I don't have an ESP32 with me, is there any workaround for it to be used with the Raspberry Pi?

**Comment by caco3** [comment]
No, the RPI is ARM based and our project is ESP32 based, there is no way this could work.
Just get an ESP32-CAM board, they are cheap.

**Comment by SybexX** [comment]
if you want to use rpi, you'll be more likely to get there with ChatGPT
